docs/development_environment.mdx

```diff
@@ -88,10 +88,15 @@ After WSL is restarted, the mount's uid and gid will match the default user.
 Install [Homebrew](https://brew.sh/), then use that to install the dependencies:
 
 ```shell
-brew install python3 autoconf ffmpeg
+brew install python3 autoconf ffmpeg rust openssl@1.1
 ```
 
-If you encounter build issues with `cryptography` when running the `script/setup` script below, check the cryptography documentation for [installation instructions](https://cryptography.io/en/latest/installation/#building-cryptography-on-macos).
+When you encounter build issues with `cryptography` when running the `script/setup` script below, the following command should work
+(based on the [cryptography installation instructions](https://cryptography.io/en/latest/installation/#building-cryptography-on-macos)):
+
+```shell
+env VIRTUAL_ENV=venv/ LDFLAGS="-L$(brew --prefix openssl@1.1)/lib" CFLAGS="-I$(brew --prefix openssl@1.1)/include" venv/bin/pip install cryptography
```

Comment:
Actually, one needs to set these variables for the entire setup task, not just a single thing...